controller/proxy-injector/webhook.go

```diff
@@ -79,6 +78,23 @@ func Inject(api *k8s.API,
 	}
 	proxyInjectionAdmissionRequests.With(admissionRequestLabels(ownerKind, request.Namespace, report.InjectAnnotationAt, configLabels)).Inc()
 
+	//Check if Namespace has valid inject annotation
+	if isNsAnnotationValid, val := isInjectionValid(nsAnnotations); !isNsAnnotationValid {
+		log.Warnf("invalid proxy injection value \"%s\" detected at namespace \"%s\" for %s", val, request.Namespace, report.ResName())
+		if parent != nil {
+			recorder.Eventf(*parent, v1.EventTypeWarning, eventTypeSkipped, "Invalid proxy injection value \"%s\" detected at namespace %s", val, request.Namespace)
```

Comment:
This event will tied to the workload whose namespace is known to the user, so no need to explicit it here; you can just say something like "at the namespace level".
Also, I wouldn't call this `injection value` but instead `Invalid value \"%s\" for annotation \"%s\`

Reply:
@alpeb yes, that makes sense! will make the required changes